.markdownlinkcheck.jsonc

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
I suggest applying the improvements that have been made to this file and associated workflow from Freyja:

- rename the file to `markdownlinkcheck.json` (it's not actually jsonc format) and update `.github/workflows/markdown-ci.yml` to use the right file (right now I think it's also expecting this config in a `tools` directory, not at the repo root, so you'll have to move this file or change the path)
- include this file and `markdownlint.jsonc` in the `on.pull_request.paths` list
- include the pattern `^https://aka.ms` since those links frequently produce false negatives and fail the workflow

Reply:
Thanks for the suggestion, I will add the tools directory as well and move any relevant files there